.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: "9.4.0"
+
+    steps:
+      - checkout
+
+      - run:
+          name: Update submodules
+          command: git submodule update --init --recursive
+
+      - run:
+          name: Build and run tests
+          command: fastlane scan
+          environment:
+            SCAN_DEVICE: iPhone 8
+            SCAN_SCHEME: libExpecta-iOS
+
+      - store_test_results:
+          path: test_output/report.xml
+      - store_artifacts:
+          path: /tmp/test-results
```

Comment:
Are you sure about this? I don't see any documentation stating that the tests results are saved to this path. Again I see that `circle-ci` for `libextobjc` has the same path, but again if I look at the output of a [job](https://circleci.com/gh/Lightricks/libextobjc/2) I see:
```
Uploading /tmp/test-results to scan-test-results
  No artifact files found at /tmp/test-results
```